include/libopencm3/at32/f40x/memorymap.h

```diff
@@ -93,13 +93,13 @@
 #define TIM9_BASE			(PERIPH_BASE_APB2 + 0x4c00)
 #define TIM10_BASE			(PERIPH_BASE_APB2 + 0x5000)
 #define TIM11_BASE			(PERIPH_BASE_APB2 + 0x5400)
-#define ACC_BASE            (PERIPH_BASE_APB2 + 0x5800)
-#define I2C3_BASE           (PERIPH_BASE_APB2 + 0x5c00)
-#define USART6_BASE         (PERIPH_BASE_APB2 + 0x6000)
-#define USART7_BASE         (PERIPH_BASE_APB2 + 0x6400)
-#define USART8_BASE         (PERIPH_BASE_APB2 + 0x6800)
-#define I2S2_EXT_BASE       (PERIPH_BASE_APB2 + 0x6c00)
-#define I2S3_EXT_BASE       (PERIPH_BASE_APB2 + 0x7000)
```

Comment:
it would be _really_ nice not to have fixup commits in the same stack please.  It makes splitting and bisection on a long set like this awful.

Reply:
i'm planning to squash this commit with first in series anyway -- if that's okay.